jwql/website/apps/jwql/data_containers.py

```diff
@@ -1647,6 +1656,34 @@ def get_preview_images_by_rootname(rootname):
     return preview_images
 
 
+def get_detectors_by_rootname(rootname):
+    """
+    Return a list of exposures with the same rootname as the provided rootname, but
+    including all available detectors.
+    
+    Parameters
+    ----------
+    rootname : str
+        The rootname of interest (e.g.
+        ``jw86600008001_02101_00007_guider2``).
+
+    Returns
+    -------
+    detector_list : list
+        A list of images that are part of the same exposure but with all detectors.
+    """
+    detector_list = []
+    search_rootname = rootname[:25]
+    filenames = get_filenames_by_rootname(search_rootname)
+    for filename in filesnames:
```

Comment:
Caught a typo here: should be `for filename in filenames:` instead of `filesnames`

Reply:
Fixed